avaje-config/src/main/java/io/avaje/config/InitialLoader.java

```diff
@@ -106,7 +111,6 @@ void loadEnvironmentVars() {
    */
   void loadLocalFiles() {
     loadMain(RESOURCE);
-    loadViaProfiles(RESOURCE);
```

Comment:
`loadViaProfiles(RESOURCE);`

I think the removal of this should probably happen on a different PR. It also appears the method still exists?

I think the plan to unify resource loading with the newer URI stuff can be done but better as a separate PR.

Reply:
Looked like a duplicate call to me

Reply:
> Looked like a duplicate call to me

It should be in a separate PR if we think this is a bug and a duplicate call and not mixed into this PR. This PR should ONLY contain changes related to a URI loader thing and nothing else including preferably no "style changes" like rename variables etc.